starter-code/src/Main.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from "react";
+import "./App.css";
+import "bulma/css/bulma.css";
+import FoodBox from "./FoodBox";
+import listOfFoods from './foods.json'
+
+
+class Main extends Component {
+  state = {
+    theList: listOfFoods,
+    filteredList: listOfFoods,
+    todaysFoods: [],
+    nameInput: '',
+    caloriesInput: '',
+    imageInput: '',
+    searchInput: '',
+    formShowing: false
+  }
+
+  showTodaysFoods = () => {
+    return (
+      this.state.todaysFoods.map((eachFoodItem) => {
+        return (
+          <div>
+            <p>
+              {eachFoodItem.quantity}  {eachFoodItem.name}
+            </p>
+            <p>
+              total calories: {eachFoodItem.calories * eachFoodItem.quantity}
+            </p>
+
+          </div>
+        )
+      })
+    )
+  }
+
+
+  showListOfFoods = () => {
+    return (
+      this.state.filteredList.map((eachFoodItem) => {
+        return (<FoodBox whatToDoWhenIClickTheButton={this.addToMenu} theFoodForThisParticularFoodBox={eachFoodItem} />)
+      })
+    )
+  }
+
+
+  updateInput = (theEvent) => {
+    this.setState({ [theEvent.target.id]: theEvent.target.value }, () => {
+      console.log(this.state)
+    })
+
+  }
+
+  addTheFood = (e) => {
+    e.preventDefault();
+    if (this.state.nameInput === '' || this.state.caloriesInput === '' || this.state.imageInput === '') {
+      return;
+    }
+    const newFood = {};
+    newFood.name = this.state.nameInput;
+    newFood.calories = this.state.caloriesInput;
+    newFood.image = this.state.imageInput;
+    const copyOfFoodList = [...this.state.theList];
```

Comment:
here, you have to make the copy of the FilteredFoods since that is the list that you use un the method showListOfFoods, like this:
```
const copyOfFoodList = [...this.state.filteredList];
    copyOfFoodList.unshift(newFood);

    this.setState({
      filteredList: copyOfFoodList,
      nameInput: '',
      caloriesInput: '',
      imageInput: ''
    });
```